.github/workflows/ci.yml

```
@@ -0,0 +1,61 @@
+        run: cargo test
+
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker image
+        run: docker build . --file Dockerfile 
```

yamllint

[error] trailing spaces

(trailing-spaces)